tuxemon/states/choice/choice_item.py

```diff
@@ -0,0 +1,81 @@
+# SPDX-License-Identifier: GPL-3.0
+# Copyright (c) 2014-2023 William Edwards <shadowapex@gmail.com>, Benjamin Bean <superman2k5@gmail.com>
+from __future__ import annotations
+
+from collections.abc import Callable, Sequence
+from typing import Any
+
+import pygame_menu
+from pygame_menu.locals import POSITION_CENTER, POSITION_EAST
+from pygame_menu.widgets.selection.highlight import HighlightSelection
+
+from tuxemon import prepare
+from tuxemon.db import db
+from tuxemon.menu.menu import PygameMenuState
+from tuxemon.menu.theme import get_theme
+from tuxemon.tools import transform_resource_filename
+
+ChoiceMenuGameObj = Callable[[], None]
+
+
+def fix_width(screen_x: int, pos_x: float) -> int:
+    """it returns the correct width based on percentage"""
+    value = round(screen_x * pos_x)
+    return value
+
+
+def fix_height(screen_y: int, pos_y: float) -> int:
+    """it returns the correct height based on percentage"""
+    value = round(screen_y * pos_y)
+    return value
+
+
+class ChoiceItem(PygameMenuState):
+    """
+    Game state with a graphic box and some text in it.
+
+    Pressing the action button:
+    * if text is being displayed, will cause text speed to go max
+    * when text is displayed completely, then will show the next message
+    * if there are no more messages, then the dialog will close
+    """
```

Comment:
I think this docstring is a copy/paste from `ChoiceState`. We need to change it, since this class doesn't serve the same purpose as `ChoiceState`.

Reply:
Yeah, it was a big chunk of information, replaced it with a small description